### ISSUE TITLE
Access Violatin reading location 

### ISSUE THREAD

**Issue by her0mx** [OgnianM/Ghost-Injector#1]
Line: 84 PE.C
```
	uint32_t* IDT_entry = (uint32_t*)rva2phys(pe, pe->NTHeaders->OptionalHeader.DataDirectory[IMAGE_DIRECTORY_ENTRY_IMPORT].VirtualAddress);
```

I got this error:
```
Exception: Exception 0xc0000005 encountered at address 0x4a294c: Access violation reading location 0xcccccd4c
```

I gonna check why don't work and but i think are problems with getting the PE file into Memory bytes

**Comment by her0mx** [comment]
I make a mistake when put the exe path, but still with out work

**Comment by her0mx** [comment]
I made tests on Windows 11 and not work y got an error on GhostWriter.C Function GWPrepareThread i will check in others OS if works

**Comment by OgnianM** [comment]
Yeah, sorry, this repo has been unmaintained for years, I don't even have a Windows system to test it on right now.
Do keep in mind that you can only inject a 32-bit process into another 32-bit process, I never got around to adding 64 bit support.
